crates/curp/src/server/cmd_worker/mod.rs

```diff
@@ -161,7 +162,10 @@ async fn worker_as<C: Command, CE: CommandExecutor<C>, RC: RoleChange>(
                 .after_sync(vec![AfterSyncCmd::new(cmd.as_ref(), false)], entry.index)
                 .await
                 .map(|res| {
-                    let (asr, _) = res.into_iter().next().unwrap();
+                    let (asr, _) = res
+                        .into_iter()
+                        .next()
+                        .unwrap_or_else(|| unreachable!("the asr should always be Some"));
```

Comment:
Shall we use `expect` to replace `unwrap_or_else(|| unreachable!("the asr should always be Some"))`?

Reply:
done